extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.tracking.js

```diff
@@ -0,0 +1,35 @@
+define('wikia.articleVideo.featuredVideo.tracking', [], function () {
+	function getDefaultState() {
+		return {
+			wasAlreadyUnmuted: false
+		}
+	}
+
+	function track(gaData) {
+		console.log(gaData);
+	}
+
+	var state = getDefaultState();
+
+	return function (featuredVideoPlayer) {
```

Comment:
consider renaming to `playerInstance`, since it's used everywhere

Reply:
done